### PR TITLE
Corrected the user creation command  CONTRIBUTORS_QUICK_START.rst

### PR DESCRIPTION
**Pull request by KesharwaniArpita**

### CODE REVIEW

File: CONTRIBUTORS_QUICK_START.rst [hunk 1]
```diff
@@ -302,18 +302,28 @@ Setting up Breeze
 5. When you enter Breeze environment you should see prompt similar to ``root@e4756f6ac886:/opt/airflow#``. This
    means that you are inside the Breeze container and ready to run most of the development tasks. You can leave
    the environment with ``exit`` and re-enter it with just ``breeze`` command.
-   Once you enter breeze environment, create airflow tables and users from the breeze CLI. ``airflow db reset``
+   
+
+.. code-block:: bash
+
+  root@b76fcb399bb6:/opt/airflow# airflow db reset
```
Comment (jscheffl) [inline CONTRIBUTORS_QUICK_START.rst:308]: I'd rather call `airflow db init` on first use.
```suggestion
  root@b76fcb399bb6:/opt/airflow# airflow db reset
```

Comment (Lee-W) [inline CONTRIBUTORS_QUICK_START.rst:308]: I guess the suggestions should be the following?

```suggestion
  root@b76fcb399bb6:/opt/airflow# airflow db init
```

Comment (Lee-W) [reply CONTRIBUTORS_QUICK_START.rst:308]: Maybe we can add a description to this as well?

**Review by utkarsharma2 — APPROVED** [review]
It looks good to me once the @Lee-W Suggestion is merged. Thanks for the contribution @KesharwaniArpita. :)

**Comment by KesharwaniArpita** [comment]
Thanks for the review, I have made the required change.

File: CONTRIBUTORS_QUICK_START.rst [hunk 2]
```diff
@@ -302,18 +302,28 @@ Setting up Breeze
...
    the environment with ``exit`` and re-enter it with just ``breeze`` command.
-   Once you enter breeze environment, create airflow tables and users from the breeze CLI. ``airflow db reset``
+   
+
+.. code-block:: bash
+
+  root@b76fcb399bb6:/opt/airflow# airflow db init
+
+
+6. Once you enter breeze environment, create airflow tables and users from the breeze CLI. ``airflow db reset``
```
Comment (jscheffl) [inline CONTRIBUTORS_QUICK_START.rst:311]: Here is is now a bit inconsistent to the lines above.
```suggestion
6. Once you enter breeze environment, create airflow tables and users from the breeze CLI. ``airflow db init``
```

**Review by jscheffl — APPROVED** [review]

**Comment by jens-scheffler-bosch** [comment]
FYI Pipeline fails but unfortunately (at least for me) the logs do not show details, error that I can see in raw logs are:

2023-10-11T15:03:03.3370615Z Remove trailing whitespace at end of line..........................................[41mFailed[m
2023-10-11T15:03:03.3371785Z [2m- hook id: trailing-whitespace[m
2023-10-11T15:03:03.3372355Z [2m- exit code: 1[m
2023-10-11T15:03:03.3373111Z [2m- files were modified by this hook[m
2023-10-11T15:03:03.3373545Z 
2023-10-11T15:03:03.3373799Z Fixing CONTRIBUTORS_QUICK_START.rst
2023-10-11T15:03:03.3374137Z

**Comment by KesharwaniArpita** [comment]
I cannot understand what do you meant by that?

**Comment by jens-scheffler-bosch** [comment]
> I cannot understand what do you meant by that?

The PR failed build and can not be merged. In my brwoser the logs failed to display. THe log part I posted above is the error message: There are tailing spaces in the RST changes you made.

**Comment by utkarsharma2** [comment]
@KesharwaniArpita You might want to install pre-commits hooks - https://github.com/apache/airflow/blob/main/STATIC_CODE_CHECKS.rst#pre-commit-hooks

**Comment by KesharwaniArpita** [comment]
Thanks for letting me know. Can you try again?

**Comment by KesharwaniArpita** [comment]
> @KesharwaniArpita You might want to install pre-commits hooks - https://github.com/apache/airflow/blob/main/STATIC_CODE_CHECKS.rst#pre-commit-hooks

Yeah, Thank you!

File: CONTRIBUTORS_QUICK_START.rst [hunk 3]
```diff
@@ -302,18 +302,27 @@ Setting up Breeze
 5. When you enter Breeze environment you should see prompt similar to ``root@e4756f6ac886:/opt/airflow#``. This
    means that you are inside the Breeze container and ready to run most of the development tasks. You can leave
    the environment with ``exit`` and re-enter it with just ``breeze`` command.
-   Once you enter breeze environment, create airflow tables and users from the breeze CLI. ``airflow db reset``
+
+.. code-block:: bash
+
+  root@b76fcb399bb6:/opt/airflow# airflow db init
```
Comment (potiuk) [inline CONTRIBUTORS_QUICK_START.rst]:
```suggestion
  root@b76fcb399bb6:/opt/airflow# airflow db reset
```

The `init` command is deprecated. Reset is the right one.

File: CONTRIBUTORS_QUICK_START.rst [hunk 4]
```diff
@@ -302,18 +302,27 @@ Setting up Breeze
...
+
+
+6. Once you enter breeze environment, create airflow tables and users from the breeze CLI. ``airflow db init``
```
Comment (potiuk) [inline CONTRIBUTORS_QUICK_START.rst]:
```suggestion
6. Once you enter breeze environment, create airflow tables and users from the breeze CLI. ``airflow db reset``
```

File: CONTRIBUTORS_QUICK_START.rst (hunk 3 above)
Comment (potiuk) [reply CONTRIBUTORS_QUICK_START.rst]: Not resolved

File: CONTRIBUTORS_QUICK_START.rst (hunk 4 above)
Comment (potiuk) [reply CONTRIBUTORS_QUICK_START.rst]: neither this one.

File: CONTRIBUTORS_QUICK_START.rst (hunk 3 above)
Comment (jscheffl) [reply CONTRIBUTORS_QUICK_START.rst]: And sorry, partly my fault - initial review from me was to change from `reset`to `init` - did not know that init is deprecated :-(

Comment (KesharwaniArpita) [reply CONTRIBUTORS_QUICK_START.rst]: No issue

**Review by potiuk — APPROVED** [review]

**Pull request merged**